docs/content/error/$controller/ctrlreg.ngdoc

```diff
@@ -0,0 +1,24 @@
+@ngdoc error
+@name $controller:ctrlreg
+@fullName A controller with this name is not registered.
+@description
+
+This error occurs when the {@link ng.$controller `$controller()`} service is called
+with a string that does not match any of the registered controllers. The controller service may have
+been invoked directly, or indirectly through the {@link ng.ngController `ngController`} directive,
+or when inside a {@link angular.Module#component component} / {@link angular.Module#directive directive}
+definition (when using string notation for the controller property).
+
+Sources for this error can be:
+
+1. You have a typo in the {@link ng.ngController `ngController`} directive,
+in a {@link angular.Module#component component} / {@link angular.Module#directive directive}
+definition's controller property, or in the call to {@link ng.$controller `$controller()`}.
+2. You have not registered the controller (neither via {@link angular.Module#controller `Module.controller`}
+nor {@link ng.$controllerProvider#register `$controllerProvider.register()`}.
+3. You have a typo in the *registered* controller name.
+4. You want to use controllers defined on the `window`, but have turned off
```

Comment:
I would mention that this  feature is deprecated/not recommened.
E.g. something like `[...] defined on `window` (a feature which is deprecated and turned off by default), but [...]`.